deployments/compose/etc/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+MAX_ATTEMPTS=15
+
+attempt_counter=0
+  while ! /opt/bitnami/spark/bin/spark-submit --class org.apache.spark.sql.hive.thriftserver.HiveThriftServer2; do
+      if [ ${attempt_counter} -eq ${MAX_ATTEMPTS} ];then
+         echo "Operation got to max attempts" >&2
```

Comment:
you write this one to the std err but the attempts to the std out. choose one :)

Reply:
changed both to std out.